src/controllers/observation-controller.ts

```diff
@@ -23,39 +23,37 @@ export default class ObservationController extends AbstractController {
     this.setMainEntity(this.observations, 'observation');
 
     this.router.get('/', this.findObservations);
+    this.router.get('/aggregations', this.getAggregations);
     this.router.post('/', this.addObservation);
-
     this.router.param('id', this.checkId);
     this.router.get('/:id', this.findOne);
     this.router.delete('/:id', this.remove);
 
     return this.router;
   }
 
-  private remove = async (req: RequestWithObservation, res: Response, next: NextFunction): Promise<void> => {
-    const { observation }: { observation: Observation } = req;
+  private findObservations = async (req: RequestWithPageParams, res: Response, next: NextFunction): Promise<void> => {
```

Comment:
rename findObservations method, because we've already had findObservation method

Reply:
fixed